tests/testthat/test-spam-dataloader.R

```diff
@@ -0,0 +1,16 @@
+if (requireNamespace("testthat", quietly = TRUE)) {
+  
+  test_that("spam_dataloader works as expected", {
+    
+    loader <- spam_dataloader(download = TRUE)
```

Comment:
You probably need to update the test cases, as `spam_dataloader` is now called `spam_dataset`.